app/context_manager.py

```diff
@@ -0,0 +1,25 @@
+from os import remove, makedirs, path
+from types import TracebackType
+from typing import Optional, Type
+
+
+class MoveFileManager:
+    def __init__(
+            self,
+            source_file_name: str,
+            path: str
+    ) -> None:
+        self.source_file_name = source_file_name
+        self.path = path
+
+    def __enter__(self) -> None:

```

Comment:
The __enter__ method should return 'self' or another object to be used in the 'with' statement. Currently, it returns None, which might not be the intended behavior.